CHANGES_UNRELEASED.md

```diff
@@ -8,4 +8,5 @@
 
 ### ⚠️ Breaking changes ⚠️
 
-- Adds support for `entrypoint` in oauth flow APIs: consumers of `beginOauthFlow` and `beginPairingFlow` (`beginAuthentication` and `beginPairingAuthentication` in ios) are now ***required*** to pass an `entrypoint` argument that would be used for metrics. This puts the `beginOauthFlow` and `beginPairingFlow` APIs inline with other existing APIs, like `getManageAccountUrl`.  ([#3265](https://github.com/mozilla/application-services/pull/3265))
\ No newline at end of file
+- Adds support for `entrypoint` in oauth flow APIs: consumers of `beginOauthFlow` and `beginPairingFlow` (`beginAuthentication` and `beginPairingAuthentication` in ios) are now ***required*** to pass an `entrypoint` argument that would be used for metrics. This puts the `beginOauthFlow` and `beginPairingFlow` APIs inline with other existing APIs, like `getManageAccountUrl`.  ([#3265](https://github.com/mozilla/application-services/pull/3265))
```

Comment:
You mention the iOS functions in this changelog entry, but I don't see any change to them in the PR itself. Do they need to be updated here as well?

Reply:
Hmmm, the ios mention is from the entrypoint change #3265. Not sure why github thinks I changed it 🤷